datalad_tabby/io/__init__.py

```diff
@@ -9,7 +14,178 @@
 from openpyxl.worksheet.worksheet import Worksheet
 
 
-__all__ = ['xlsx2tabby', 'tabby2xlsx']
+__all__ = ['xlsx2tabby', 'tabby2xlsx', 'load_tabby']
+
+
+def load_tabby(
+    src: Path,
+    *,
+    single: bool = True,
+    jsonld: bool = True,
+) -> Dict | List:
+    """Load a tabby (TSV) record as structured (JSON(-LD)) data
+
+    The record is identified by the table/sheet file path ``src``. This need
+    not be the root 'dataset' sheet, but can be any component of the full
+    record.
+
+    The ``single`` flag determines whether the record is interpreted as a
+    single entity (i.e., JSON object), or many entities (i.e., JSON array of
+    (homogeneous) objects).  Depending on the ``single`` flag, either a
+    ``dict`` or a ``list`` is returned.
+
+    Other tabby tables/sheets are loaded when ``@tabby-single|many-`` import
+    statements are discovered. The corresponding data structures are replace
+    the import statement at its location.
```

Comment:
```suggestion
    Other tabby tables/sheets are loaded when ``@tabby-single|many-`` import
    statements are discovered. The corresponding data structures then replace
    the import statement at its location.
```
or an equivalent grammar fix